drivers/counter/counter_ll_stm32_rtc.c

```diff
@@ -52,6 +53,17 @@ LOG_MODULE_REGISTER(counter_rtc_stm32, CONFIG_COUNTER_LOG_LEVEL);
 #define RTC_EXTI_LINE	LL_EXTI_LINE_17
 #endif
 
+#if defined(CONFIG_SOC_SERIES_STM32F1X)
+#define COUNTER_NO_DATE
+#define LL_RTC_ALMA_Init(...) LL_RTC_ALARM_Init(__VA_ARGS__)
+#define LL_RTC_ClearFlag_ALRA(...) LL_RTC_ClearFlag_ALR(__VA_ARGS__)
+#define LL_RTC_EnableIT_ALRA(...) LL_RTC_EnableIT_ALR(__VA_ARGS__)
+#define LL_RTC_DisableIT_ALRA(...) LL_RTC_DisableIT_ALR(__VA_ARGS__)
+#define LL_RTC_IsActiveFlag_ALRA(...) LL_RTC_IsActiveFlag_ALR(__VA_ARGS__)
+#define LL_RTC_ALMA_Enable(...) {}
+#define LL_RTC_ALMA_Disable(...) {}
+#endif
```

Comment:
I reckon this method is pretty efficient for readability, but unfortunately generally renaming LL/HAL APIs should be avoided to avoid confusion for users/readers.
Instead we recommend creating a new set of functions which will provide abstraction. For instance see what is done for [STM32 SPI](https://github.com/zephyrproject-rtos/zephyr/blob/b2f80b0e16363021bc1fd65884b11830fa869870/drivers/spi/spi_ll_stm32.h#L85).

Reply:
I changed to inline function.

Reply:
Perfect, thanks